src/main/java/org/traccar/notificators/NotificatorPushover.java

```diff
@@ -56,25 +56,24 @@ public NotificatorPushover(Config config, NotificationFormatter notificationForm
         url = "https://api.pushover.net/1/messages.json";
         token = config.getString(Keys.NOTIFICATOR_PUSHOVER_TOKEN);
         user = config.getString(Keys.NOTIFICATOR_PUSHOVER_USER);
-        if (token == null || user == null) {
-            throw new RuntimeException("Pushover token or user missing");
-        }
     }
 
     @Override
     public void send(User user, Event event, Position position) {
-
-        String device = "";
-        if (user.hasAttribute("notificator.pushover.device")) {
-            device = user.getString("notificator.pushover.device").replaceAll(" *, *", ",");
-        }
-
         var shortMessage = notificationFormatter.formatMessage(user, event, position, "short");
 
         Message message = new Message();
         message.token = token;
-        message.user = this.user;
-        message.device = device;
+
+        message.user = user.getString("pushoverUserKey");
```

Comment:
I think we should reuse `NOTIFICATOR_PUSHOVER_USER` here.

Reply:
I've used the [NotificatorTelegram.java ](https://github.com/lzkill/traccar/blob/ea68646e6572945db82bfdf8e732957492e19b03/src/main/java/org/traccar/notificators/NotificatorTelegram.java)as a guide, where the config provides a wide value that can be overwritten by an user attribute. (`NOTIFICATOR_TELEGRAM_CHAT_ID` / `telegramChatId`)

Reply:
We can't really change what's already there, but we can enforce this for new attributes.